CHANGELOG_DESKTOP.md

```diff
@@ -1,5 +1,39 @@
 # Changelog
 
+## [1.41.96](https://github.com/brave/brave-browser/releases/tag/v1.41.96)
+
+ - Added Solana Dapp support for Brave Wallet. ([#23520](https://github.com/brave/brave-browser/issues/23520))
```

Comment:
We've been using `Dapp` since the beginning as per https://raw.githubusercontent.com/brave/brave-browser/master/CHANGELOG_DESKTOP.md. If that's the case, we'll need to change all instances of `Dapp` -> `DApp`.

Reply:
as discussed, will change in a follow up 👍🏻